src/mobile/apps/partner_profile/client/artworks.coffee

```diff
@@ -27,7 +27,9 @@ module.exports.PartnerArtworksView = class PartnerArtworksView extends Backbone.
           @$('#partner-artworks-spinner').hide()
 
   render: =>
-    @$('#partner-artworks-list ul').append $(artworkColumnsTemplate(artworkColumns: @collection.groupByColumnsInOrder())).html()
+    $columns = $(artworkColumnsTemplate(artworkColumns: @collection.groupByColumnsInOrder()))
+    @$('#partner-artworks-list .artwork-columns-column').each (i, column) =>
+      $(column).append $columns.find('.artwork-columns-column').eq(i).html()
```

Comment:
Note that this naively appends grouped artworks to column with the same index, so it's not guaranteed the height of the columns would be balanced. We can follow up separately if necessary.